cerner/cds-hooks-sandbox
# Fix text runoff on card demo view on card rendering. Add img html tag for placing specified icons on cards in 100x100 format

Issues: https://github.com/cds-hooks/sandbox/issues/81, https://github.com/cds-hooks/sandbox/issues/74

Currently, if a large amount of text is present on the card demo page (see 2nd issue above), the text will run off the page on the card render in the Card Demo view of the Sandbox. Since the container wrapping the main components on that view is a `table` display, we should switch the child components to be of a `table-cell` display, and use `table-layout: fixed` for the top parent component to base the width off the table's width (100%). This should keep both the card render space and the text box space at 50% width (no run off).

Additionally, the Sandbox does not currently have an HTML element to display the icon on a Card as it renders, thus ignoring the `source.icon` property from a CDS response. We should add an `img` tag at a fixed 100x100 size according to the spec, and display a message in its place if the Sandbox cannot fetch the image.

To test:
- Use text from 2nd issue listed above in the card demo view to ensure text does not run off
- Additionally, add an icon in the card by adding an `icon` property under the `source` object. You can use https://content.hspconsortium.org/images/my-web-app/logo/lightbulb_100x100.png to see it render

@kpshek 
@mjhenkes 
@kolkheang
